sdks/java/core/src/test/java/org/apache/beam/sdk/testing/PAssertTest.java

```diff
@@ -385,10 +385,11 @@ public void testContainsInAnyOrderFalse() throws Exception {
   @Category(RunnableOnService.class)
   public void testEmptyFalse() throws Exception {
     PCollection<Long> vals = pipeline.apply(CountingInput.upTo(5L));
-    PAssert.that(vals).empty();
+    PAssert.that("Vals should have been empty", vals).empty();
```

Comment:
Add the same to a view-based assertion; I believe we already have tests for them.

It's also would be nice to ensure that by default the name of the input PCollection is present.